build.rs

```diff
@@ -3,6 +3,7 @@ use rustc_version::{version, Version};
 fn main() {
     println!("cargo::rustc-check-cfg=cfg(RUSTC_LINT_REASONS_IS_STABLE)");
     println!("cargo::rustc-check-cfg=cfg(RUSTC_NEW_UNINIT_IS_STABLE)");
+    println!("cargo::rustc-check-cfg=cfg(RUSTC_HAS_UNSAFE_PINNED)");
```

Comment:
please use `CONFIG_RUSTC_HAS_UNSAFE_PINNED`, then I don't need to rename it when the kernel also wants to add support.